fvm/fvm_bench_test.go

```diff
@@ -460,130 +523,288 @@ func BenchmarkRuntimeTransaction(b *testing.B) {
 	}
 
 	templateTx := func(rep int, prepare string) string {
-		return fmt.Sprintf(`
+		return fmt.Sprintf(
+			`
 			import FungibleToken from 0x%s
 			import FlowToken from 0x%s
 			import TestContract from 0x%s
+			import EVM from 0x%s
```

Comment:
would this import affect non-evm tests? would it make sense to only have this for fvm benchmarks?

Reply:
It does, but the important part of these tests is the comparative performance.

We can deduct the cost of that import by looking at the `reference_tx` which is basically a (almost) no-op transaction

Reply:
ohh nice, then it doesn't matter yeah